rclcpp/test/rclcpp/topic_statistics/test_subscription_topic_statistics.cpp

```diff
@@ -277,12 +275,12 @@ class MessageWithHeaderSubscriber : public rclcpp::Node
 class TestSubscriptionTopicStatisticsFixture : public ::testing::Test
 {
 protected:
-  void SetUp()
+  void SetUp() override
   {
     rclcpp::init(0 /* argc */, nullptr /* argv */);
   }
 
-  void TearDown()
+  void TearDown() override
```

Comment:
Is there a reason this is being overridden?

Reply:
Well, because to properly override virtual method from the base class.